src/Loading/Loading.js

```diff
@@ -0,0 +1,14 @@
+import './Loading.css';
+import { NavLink } from 'react-router-dom'
+
+const Loading = () => {
+
+  return (
+      <div className='loading'>
+        <h1>Page is Loading...</h1>
+        {/* <NavLink to='/' className='go-home'>🏕️ go back to home 🏕️</NavLink> */}
```

Comment:
can probably remove this commented out code now? Unless there was something else you were wanting to try with it?